background.js

```diff
@@ -18,4 +18,12 @@ browser.runtime.onMessage.addListener((message, sender, sendResponse) => {
 
     return true;
   }
+  else if (message.cmd === "protectionError") {
```

Comment:
To be checked with @gurumukhi, but coding style (seems to be K&R-like) wants `else` or `else if` in the same line than closing bracket.
This is of course a minor detail.